### DISCUSSION TITLE
GLPI Inventory Task actors missing

### DISCUSSION THREAD

**Discussion by walterdasilva** [glpi-project/glpi-agent#633] [Q&A]
### Bug reporting acknowledgment

Not applicable

### Professional support

Still not applicable

### Describe the bug

GLPI Inventory Task are missing actors set. Agent still running on device normally. after some time computer missing from assets computers and of course from glpi inventory task. assets return after next agent inventory but inventory task actors still missed

### To reproduce

install and set glpi inventory

### Expected behavior

glpi inventory task actors not missing

### Operating system

Linux

### GLPI Agent version

1.7.1

### GLPI version

10.0.13

### GLPIInventory plugin or other plugin version

GLPI Inventory v1.3.5

**Comment by g-bougard** [comment]
Hi @walterdasilva 

as you're not describing a glpi-agent bug, but a symptom in GLPI, I have to convert your issue as a Q&A discusison where I'll try to give few some clues to understand what's going on.

**Comment by g-bougard** [comment]
> after some time computer missing from assets computers

I'm really not sure what you mean by this sentence. A computer can only be removed from the asset list manually or by a cleaning automatic action. So if it's really the case, check enabled automatic actions and related policy.

Maybe you indeed mean the computer agent is missing from agent list ?

In that case, you may have another agent running on the computer (like FI-Agent) or you may have run manually another agent. And this other agent just replaced the one you expect. Otherwise you may also had a cleaning automatic action that removed that agent from the list. In that last case, you may have also to review and change that agents cleaning automatic action policies.

I hope this helped you.

**Comment by walterdasilva** [comment]
Hi @g-bougard 

I've just check your directions. Yes, computer agent is missing from agent list and computer is missing from assets same time. i reaaly don´t ran nothing manually and there is only glpi-agent running on this pc.

i've just disable cleanoldagent automatic action . but there is something more. i forced inventory on this pc and nothing happend.

Regards,

**Comment by walterdasilva** [comment]
Hi @g-bougard ,

I've just check your directions. Yes, computer agent is missing from agent list and computer is missing from assets same time. i reaaly don´t ran nothing manually and there is only glpi-agent running on this pc.

for now i just disable cleanoldagent.

for now the agent can not show up on agents or the pc do not show on assets.

**Comment by g-bougard** [comment]
Are you using plugins like datainjection or uninstall ? Particularly these ones as they can have an impact on asset management.

Check also if the asset is not available in trashbin. If yes, you may see in its history why or how the asset came there.

**Comment by walterdasilva** [comment]
Hi @g-bougard ,

First, no there is nothing in trashbin.

And Yes, i'was using uninstalling plugin. so not using anymore.

but inventory task still missing actors information and assets still missing pc records and trashbin still empty
